lib/admin/components/UserSettings.js

```diff
@@ -276,56 +276,61 @@ export default class UserSettings extends Component<Props, State> {
               })}
             </SplitButton>
           </h3>
-        }
-      >
-        {orgProjects.map((project: Project, i: number) => {
-          if (i !== this.state.currentProjectIndex) return null
-          const settings = this.state.projectSettings[project.id]
-          return <ProjectAccessSettings
-            project={project}
-            key={project.id}
-            settings={settings}
-            fetchProjectFeeds={fetchProjectFeeds}
-            visible={i === this.state.currentProjectIndex}
-            projectAccessUpdated={this.projectAccessUpdated}
-            projectFeedsUpdated={this.projectFeedsUpdated}
-            projectPermissionsUpdated={this.projectPermissionsUpdated} />
-        })}
+        </Panel.Heading>
+
+        <Panel.Body>
+          {orgProjects.map((project: Project, i: number) => {
+            if (i !== this.state.currentProjectIndex) return null
+            const settings = this.state.projectSettings[project.id]
+            return <ProjectAccessSettings
+              project={project}
+              key={project.id}
+              settings={settings}
+              fetchProjectFeeds={fetchProjectFeeds}
+              visible={i === this.state.currentProjectIndex}
+              projectAccessUpdated={this.projectAccessUpdated}
+              projectFeedsUpdated={this.projectFeedsUpdated}
+              projectPermissionsUpdated={this.projectPermissionsUpdated} />
+          })}
+        </Panel.Body>
       </Panel>
     )
     return (
       <Row>
         <Col xs={4}>
-          <Panel header={<h3>Organization settings</h3>}>
-            {creatorIsApplicationAdmin &&
+          <Panel>
+            <Panel.Heading><h3>Organization settings</h3></Panel.Heading>
```

Comment:
See comment about `<h3>` above.

Reply:
Removed in 5353f432defd32577a1378e0857c2946989d4bcd